src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -1,18 +1,15 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-import logging
 import torch
 import torch.nn as nn
 from nni.compression.torch.compressor import PrunerModuleWrapper
 
-
-_logger = logging.getLogger(__name__)
-
 try:
     from thop import profile
-except ImportError:
-    _logger.warning('Please install thop using command: pip install thop')
+except Exception as e:
+    print('Please install thop using command: pip install thop')
```

Comment:
'Please install thop using command: pip install thop' -> 'thop is not found, please install the python package: thop'